plugins/operators/model_evaluation/__init__.py

```
@@ -0,0 +1,1052 @@
+            invalid=True,
+        )
+
+    def render_use_custom_code_warning(
+        self, inputs, reason=CustomCodeViewReason.TOO_MANY_CATEGORIES
+    ):
+        label, description = None, None
+
+        if reason == CustomCodeViewReason.TOO_MANY_CATEGORIES:
+            label = "Too many categories."
+            description = (
+                f"Selected field has too many values to display. "
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)